jim-munemitsu.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+
+<html>
+
+  <head>
+    <title>CSS Timeline</title>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <link rel="stylesheet" href="css/jim-munemitsu.css">
+  </head>
+
+  <body>
+    <h1>Brief History of CSS</h1>
+      <ol>
+        <li id="first"><span class="date">1994</span>: The first work on style sheets started at CERN.</li>
+        <li><span class="date">December 1996</span>: CSS was proposed as W3C standard:</li>
+        <blockquote>
+          "CSS1 defines the language in which style sheets are written and a set of properties that designers can use to describe how documents should be presented. The properties in CSS1 cover the most common screen-based presentations."
+        </blockquote>
+        <li><span class="date">November 1997</span>: The W3C released the first public CSS2 Working Draft.</li>
+
+          <ul>
+              <p><strong>New features:</strong></p>
+
+                <li>Ability to specify how a document should be spoken when rendering HTML with a speech synthesizer.</li>
+                <ul>
+                  <li>You can specify how loudly a phrase should be spoken, where pauses should be, voice characteristics, and where one speaker is physically in relation to others (for stereo effects.)</li>
+                  <li>You can insert sound clips before and after elements, for instance, to emphasize hypertext links.</li>
+                </ul>
+            <li>More complex positioning of text, images and other elements on the page.</li>
+
+              <ul>
+                <li>You can position text in a box within a paragraph.</li>
+                <li>You can position and "layer" material as though it was written on acetate sheets one on top of each other: most kinds of layout which are commonly found in magazines and other printed material are possible with CSS2.</li>
+              </ul>
+
+            <li>You can specify styles for printing and other media.</li>
+              <ul>
+                <li>Stylesheet for printing on paper may need to be different from one for presenting information on a screen.</li>
+                <li>CSS2 lets authors associate different stylesheets for different media with the same document.</li>
+                <li>Authors can control page breaks.</li>
+                <li>Has widow and orphan control, just as for ordinary desktop publishing.</li>
+                <li>For high-quality printing, crop marks can be printed outside the outer edges of the page box.</li>
+              </ul>
+            <li>Better specification of fonts for a document.</li>
+              <ul>
+                <div>
```

Comment:
See the [permitted content](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) section on MDN. `<div>` is not allowed in `<ul>`.